www/src/pages/playground.tsx

```diff
@@ -35,20 +46,54 @@ export default function Playground({ location }) {
     return () => clearInterval(iframeUrlPoll);
   }, []);
 
+  useEffect(() => {
+    setCopyUrlState('default');
+  }, [location.href]);
+
   return (
     <div className="d-flex flex-column w-100 vh-100 m-0 p-0">
       <SEO title="Playground" />
       <div className="pgn-doc__header py-3 bg-dark text-white sticky-top">
-        <div className="d-flex align-items-center justify-content-center">
+        <div className="d-flex align-items-center justify-content-end px-4" style={{ gap: '16px' }}>
+          <Button
+            variant="inverse-tertiary"
+            onClick={() => {
+              playroomStorage.clear().then(() => {
+                iframeRef!.current!.contentWindow!.location.search = '?code=N4XyA';
```

Comment:
What is `?code=N4XyA`? Maybe it`s the initial value in the URL Playroom?

Reply:
Moved it into a separate constant to explain it's value